File: python/FizzBuzz.py
#!/usr/bin/python
"""
 Fizz Buzz in python 3

 P Campbell
 February 2018
"""

for i in range(1,101):
    if i % 3 == 0 or i % 5 == 0 :
        if i % 3 == 0:
            msg = "Fizz"
        if i % 5 == 0:
            msg += "Buzz"
        print (msg)
        msg = ""
    else:
        print (i)

File: ruby/FizzBuzz.rb
#!/usr/bin/ruby
=begin
  P.M.Campbell
  2020-02-28
  Fizz Buzz challenge
=end

for i in 1..100
  if i % 3 == 0 || i % 5 == 0  
      if i % 3 == 0 
        msg = "Fizz"
      end
      if i % 5 == 0
        msg += "Buzz"
      end
      puts  msg
      msg = ""
  else 
    puts i
  end
end
